### ISSUE TITLE
Double-quoted column names are not queryable

### ISSUE THREAD

**Issue by blampe** [sqlc-dev/sqlc#2676]
### Version

1.20.0

### What happened?

Running sqlite's `.schema` will produce output with double-quoted table and column names, but those columns are incorrectly reported as not existing.

Single-quoting the column is not a valid workaround because it produces a string literal:

```
sqlite> CREATE TABLE Test ("Id" INTEGER);
sqlite> INSERT INTO Test VALUES (1);
sqlite> SELECT Id, "Id", `Id`, 'Id' FROM TEST;
1|1|1|Id
```

Note that sqlc doesn't handle any of the valid identifiers -- [Id](https://play.sqlc.dev/p/c5e9fd5fbcfa9698948e84ad720d40f061b2eeaba674f036a343d9cac6bc3285),  ["Id"](https://play.sqlc.dev/p/945fd5d6cfaf5c1991d072facc3c2e1c77ee817d5484fa99111530f6fdbc2ae6), or [\`Id\`](https://play.sqlc.dev/p/fb290d734e1158168aa2e4592e14c9d1a29ef457d1940ac9ffe98b0124f56f4b).
### Relevant log output

```shell
query.sql:6:8: column "id" does not exist
```

### Database schema

```sql
CREATE TABLE Test (
    "Id" INTEGER
);
```

### SQL queries

```sql
SELECT Id FROM Test;
```

### Playground URL

https://play.sqlc.dev/p/c5e9fd5fbcfa9698948e84ad720d40f061b2eeaba674f036a343d9cac6bc3285

### What operating system are you using?

macOS

### What database engines are you using?

SQLite

### What type of code are you generating?

Go

**Comment by blampe** [comment]
> Please stop using double quotes in CREATE TABLE or use `SELECT 'Id' FROM Test`.
> https://play.sqlc.dev/p/c4b0199a0fb88520f2cb9410be15db13c6a1f1d00b31c11c60f0f4f3aae25e02
> https://play.sqlc.dev/p/4ebaca4b8682b6d5b456155832b58714834fa2014771ae236751fd847bab73d0

@orisano `'Id'` (with single quotes) is a [string literal](https://sqlite.org/lang_keywords.html#). The alternative query you provided does not work. 

For example:

```
sqlite> CREATE TABLE Test ("Id" INTEGER);
sqlite> INSERT INTO Test VALUES (1);
sqlite> SELECT Id, "Id", 'Id' FROM TEST;
1|1|Id
```

> Please stop using double quotes in CREATE TABLE

This is the default behavior of sqlite's built-in `.schema` command:
```
sqlite> .schema
CREATE TABLE Test ("Id" INTEGER);
```

`.schema` is essential for migrating an existing database to `sqlc`. It's also a very reasonable developer workflow to incrementally modify a database and then re-generate a schema for sqlc with `.schema` when finished.

The request is to handle sqlite's default behavior gracefully, instead of requiring developers to post-process the output of `.schema` in order for sqlc to consume it.

**Comment by orisano** [comment]
> @orisano 'Id' (with single quotes) is a [string literal](https://sqlite.org/lang_keywords.html#). The alternative query you provided does not work.

Sorry, I made a mistake.
`SELECT "Id" FROM Table` works correctly with HEAD. (Not released yet).

**Comment by blampe** [comment]
> > @orisano 'Id' (with single quotes) is a [string literal](https://sqlite.org/lang_keywords.html#). The alternative query you provided does not work.
> 
> Sorry, I made a mistake. `SELECT "Id" FROM Table` works correctly with HEAD. (Not released yet).

Amazing! I will give it a try, thank you!

**Comment by kyleconroy** [comment]
This is now fixed as of v1.22.0
